### PR TITLE
Global css vars

### PR DESCRIPTION
**Pull request by VadimMikh**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--edge-delivery-ubs-faweb-demo--netcentric.aem.page/
- After: https://global-css-vars--edge-delivery-ubs-faweb-demo--netcentric.aem.page/

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -11,44 +11,127 @@
...
+  --col-neutral30: #CCCABC;
+  --col-neutral40: #B8B3A2;
+  --col-neutral50: #8E8D83;
+  --col-neutral60: #7A7870;
+  --col-neutral70: #5A5D5C;
+  --col-neutral80: #404040;
+  --col-neutral90: #1C1C1C;
+
+  /* Background Set */
+  --col-bg-01: #F9F9F7;
```
Comment (dragoss-nc) [inline styles/styles.css]: I would say it's better to use the neutral color variables here, for example 

```css
--col-bg-01: var(--col-neutral10)
```

File: styles/styles.css [hunk 2]
```diff
@@ -11,44 +11,127 @@
...
+
+  /* Grey Light Set */
+  --col-167: #E6E6E6;
+  --col-176: #D7D7D7;
+  --col-177: #D3D3D3;
+  --col-178: #CCC;
+  --col-179: #FAFAFA;
+
+  /* Grey Mid Set */
+  --col-23: #1C1C1C;
```
Comment (dragoss-nc) [inline styles/styles.css]: This is identical to --col-neutral90. Do we need it?

File: styles/styles.css [hunk 3]
```diff
@@ -11,44 +11,127 @@
...
+  --col-26: #444;
+  --col-28: #646464;
+  --col-29: #919191;
+  --col-49: #717171;
+  --col-50: #333;
+
+  /* Other colours */
+  --col-30: #BEBEBE;
+  --col-168: #B30000;
+  --col-169: #DA0000;
```
Comment (dragoss-nc) [inline styles/styles.css]: This is the same as --col-brand40. Do we need it?

File: styles/styles.css [hunk 4]
```diff
@@ -11,44 +11,127 @@
...
+  --col-84: #78CAE7;
+  --col-35old: #6a7d39;
```
Comment (dragoss-nc) [inline styles/styles.css]: Maybe we could rename these 2. I know you c/p from the current project, but I never liked them.

File: styles/styles.css (hunk 1 above)
Comment (kesiah) [inline styles/styles.css]: I would just keep here `col-neutral` and `col-brand` for now.

File: styles/styles.css (hunk 2 above)
Comment (kesiah) [reply styles/styles.css]: please remove them, they are old CSS variables for communities.

File: styles/styles.css [hunk 5]
```diff
@@ -11,44 +11,127 @@
...
+  /* max width breakpoint */
+  --b-767: 767px;
+  --b-1279: 1279px;
+
+  /* min-width aliases. */
+  --bp-s: var(--b-768);
+  --bp-l: var(--b-1280);
+
+  /* max-width aliases. */
+  --bp-s-max: var(--b-767);
```
Comment (kesiah) [inline styles/styles.css]: remove max-width, we should avoid it

File: styles/styles.css [hunk 6]
```diff
@@ -11,44 +11,127 @@
...
+  --spacing-component-main-xs: 20px;
+  --spacing-component-main-sm: 32px;
+  --spacing-component-main-xl: 40px;
+
+  /* Shadows */
+  --box-shadow: 0 1px 2px 0 rgba($col-29, 0.34), 0 0 13px 0 $col-167;
+
+  /* colors */
+  --background-color: var(--col-white);
+  --light-color: var(--col-neutral10);
```
Comment (kesiah) [inline styles/styles.css]: Confusing name for me, I don't know what to expect, if light brand color or light neutral color.

File: styles/fonts.css
Comment (kesiah) [inline styles/fonts.css:1]: we can remove roboto font completely

**Review by dragoss-nc — APPROVED** [review]

File: styles/styles.css [hunk 7]
```diff
@@ -11,62 +11,95 @@
...
+  --grid-max-width: 1290px;
+  --grid-container-width-xs: 20px;
+  --grid-container-width-sm: 42px;
+  --grid-container-width-xl: 75px;
+  --spacing-component-main-xs: 20px;
+  --spacing-component-main-sm: 32px;
+  --spacing-component-main-xl: 40px;
+
+  /* Shadows */
+  --box-shadow: 0 1px 2px 0 rgba($col-29, 0.34), 0 0 13px 0 $col-167;
```
Comment (kesiah) [inline styles/styles.css]: `$col-29`? $col-167?

File: styles/styles.css [hunk 8]
```diff
@@ -11,62 +11,95 @@
...
 
+/* fallback fonts */
 @font-face {
-  font-family: roboto-fallback;
+  font-family: frutiger-fallback;
   size-adjust: 99.529%;
   src: local('Arial');
 }
 
 @media (width >= 900px) {
```
Comment (kesiah) [inline styles/styles.css]: Remove this block. We are not going to use that.

File: styles/styles.css [hunk 9]
```diff
@@ -11,62 +11,95 @@
...
 :root {
-  /* colors */
-  --background-color: white;
-  --light-color: #f8f8f8;
-  --dark-color: #505050;
-  --text-color: #131313;
-  --link-color: #3b63fb;
-  --link-hover-color: #1d3ecf;
-
   /* fonts */
```
Comment (kesiah) [inline styles/styles.css:25]: To define fonts, I would suggest using something similar to:
`ubs-events-ticketmgmt/ubs-events-ticketmgmt-frontend-apps/ubs-ticketmgmt/src/styles/variables.ts`

We create CSS variables for every font defined in Figma  (size. and line height) and also update the size of it in the breakpoints, if needed.

This way, we have a clear idea of which font we are using, and the name corresponds to the font name given in Figma also. 

WDYT?

```
:root {
      --hl20-font-size: XXpx;
      --hl20-line-height: XXpx;

      --hl22-font-size: XXpx;
      --hl2-line-height: XXpx;

      --txt141-font-size: XXpx;
      --txt141-line-height: XXpx;

      --txt150-font-size: XXpx;
      --txt150-line-height: XXpx;
     
     ........

     @media (min-width: 1200px) {
        --hl20-font-size: XXpx;
        --hl20-line-height: XXpx;

        --hl22-font-size: XXpx;
        --hl22-line-height: XXpx;

        --txt150-font-size: XXpx;
        --txt150-line-height: XXpx;

        ........
     }
}
```

**Review by kesiah — APPROVED** [review]

**Pull request merged**